### PR TITLE
Merge branch 'master' of https://github.com/lenshuygh/McJava into MCJ_29

### PR DESCRIPTION
**Pull request by Fonthom**
error line 18 in controller class when put in other package.

### CODE REVIEW

File: src/be/mcjava/fxml/CustomerScreenController.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CustomerScreenController {
+    
+    @FXML
+    private void continuePressed(ActionEvent event) throws Exception{
```
Comment (lenshuygh) [inline src/be/mcjava/fxml/CustomerScreenController.java]: the name of this method is very non-specific, i suggest naming it something like continueFromLoginToMenuScreen...
when the method name is clear the code reads a lot more easy

**Review by lenshuygh — CHANGES_REQUESTED** [review]
- you got rid of the checking of the entered data, why?
- is it possible to change the package name from 'fxml' to 'view' and put the controller in a 'controller' package (under 'be.mcjava')?  I remember something on slack about it but can't find it....
- please also change the 'menuTextAndImages' to lowercase
- can we move the 'mcDoLogo' into the menuTextAndImages folder, i think thats the right place for it

**Comment by Fonthom** [comment]
When Controller class is in designated package it gives an error message at this line "Parent root1 = fxmlLoader.load();". The only way I could make it work is in here.

**Review by lenshuygh — APPROVED** [review]

**Review by ThomassenCathleia — APPROVED** [review]

**Comment by ThomassenCathleia** [comment]
> When Controller class is in designated package it gives an error message at this line "Parent root1 = fxmlLoader.load();". The only way I could make it work is in here.

I've looked at the error and that's because the CustomerScreen2.fxml is not in the same package as the controller.

If you change 
`FXMLLoader fxmlLoader = new FXMLLoader(getClass().getResource("CustomerScreen2.fxml"));`
to
`FXMLLoader fxmlLoader = new FXMLLoader(getClass().getResource("../view/CustomerScreen2.fxml"));`
it will work again.

**Pull request merged**